package.json

```diff
@@ -26,34 +26,32 @@
     },
     "homepage": "https://github.com/fluid-project/uio-plus#ui-options-plus-uio",
     "scripts": {
-        "lint": "grunt lint",
-        "pretest": "node node_modules/rimraf/bin.js reports/* coverage/* && grunt build",
-        "test": "npm run test:node && npm run test:browser",
-        "test:node": "node node_modules/nyc/bin/nyc.js node tests/node/all-tests.js",
-        "test:browser": "node node_modules/testem/testem.js ci --file tests/browser/testem.js",
-        "test:vagrant": "vagrant up && vagrant ssh -c 'cd /home/vagrant/sync/; DISPLAY=:0 npm test'",
-        "test:vagrantBrowser": "vagrant up && vagrant ssh -c 'cd /home/vagrant/sync/; DISPLAY=:0 npm run test:browser'",
-        "test:vagrantNode": "vagrant up && vagrant ssh -c 'cd /home/vagrant/sync/; DISPLAY=:0 npm run test:node'",
-        "posttest": "node node_modules/nyc/bin/nyc.js report --reporter text-summary --reporter html"
+        "lint": "fluid-lint-all",
+        "pretest": "run-s -l clean:test",
+        "test": "run-s test:node test:browser",
+        "test:node": "nyc node tests/node/all-tests.js",
+        "test:browser": "testem ci --file tests/browser/testem.js",
+        "test:report": "nyc report -r text-summary -r html -r json",
+        "posttest": "npm run test:report",
+        "clean": "run-p clean:*",
+        "clean:build": "rimraf dist",
+        "clean:test": "rimraf reports coverage",
+        "build": "run-s -l clean:build build:*",
+        "build:dist": "cpy . ../dist --cwd=src; cpy . ../../dist/lib/infusion --cwd=node_modules/infusion"
```

Comment:
I think the ; operator is non-portable - I get an error running this line on Windows, replacing it with & allows the build to complete